src/app/books.ts

```diff
@@ -1070,7 +1070,7 @@ export const BOOKS : Book[] = [
 		]
 	}
 	,{
-		"id": 38,
+		"id": 82,
```

Comment:
Bug fix with detail pages - ID 38 was listed twice. Gave one of them a unique, unused number.